### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by val-smyrnova-datrycs** [fyrst-digital/elysium#180]
Verison 4.9
Shopware 6.7.14.1

There are several accessibility issues, would be nice also to have them fixed

Ensure role attribute has an appropriate value for the element. 
`<section id="elysiumSlider-01a002..." class="cms-element-elysium-..." role="region" data-elysium-slider="" data-swiper-options="{"effect":"slide","f...">`

Ensure every ARIA button, link and menuitem has an accessible name
`<span class="swiper-pagination-bullet" role="button" tabindex="0"></span>`

Ensure aria-hidden elements are not focusable nor contain focusable elements
`<li role="listitem" class="swiper-slide swiper-slide-prev" tabindex="0" style="width: 1512px;" aria-hidden="true">`

<img width="1300" height="507" alt="Image" src="https://github.com/user-attachments/assets/06dfc4ba-276a-4cc4-8013-24c66f311012" />
<img width="991" height="523" alt="Image" src="https://github.com/user-attachments/assets/2424bb1a-d44f-46ea-a516-3e165cb72bbf" />
<img width="1434" height="724" alt="Image" src="https://github.com/user-attachments/assets/2073e2b1-8e48-4205-b9c7-4e37db94f4ae" />
<img width="977" height="503" alt="Image" src="https://github.com/user-attachments/assets/5679d418-0893-46bd-99a1-41bb8b9b5669" />
<img width="1003" height="493" alt="Image" src="https://github.com/user-attachments/assets/e958ceea-71a2-4c89-baf0-1648347a05ca" />
